pkg/configmap/configmap.go

```diff
@@ -98,6 +98,35 @@ func buildLabelKeysCopy(labelKeys []kueue.LabelKeys) []string {
 	return ret
 }
 
+func buildManagedJobsWithoutQueueName(workloadManagement kueue.WorkloadManagement) bool {
+	if workloadManagement.LabelPolicy == "" {
```

Comment:
This if statement is superfluous, since the boolean comparison on 105 catches it.

Reply:
updated.